packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -180,6 +180,11 @@ Posts.getLastCommentedAt = (post) => {
   }
 }
 
+Posts.getLastCommentPromotedAt = (post) => {
+  return post.lastCommentPromotedAt;
+  // NOTE: doesn't yet have special casing for alignment forum
```

Comment:
Hmm, does this mean this will kind of just break horribly for AIAF posts? That seems kind of bad.